### DISCUSSION TITLE
What RV version is OpenRV 1.0 based on?

### DISCUSSION THREAD

**Discussion by faulknermano** [AcademySoftwareFoundation/OpenRV#97] [General]
Which commercial RV version was OpenRV 'branched' from? Thank you!

**Comment by compaga** [comment]
Hi faulknermano, the real answer is none. The upcoming release of commercial RV will be kind of aligned with Open RV but not built on top of Open RV. We are planning a better alignment later this year.

**Reply by faulknermano** [reply]
Thanks for the reply, Alain!
